packages-content-model/roosterjs-content-model-core/lib/override/pasteCopyBlockEntityParser.ts

```diff
@@ -0,0 +1,37 @@
+import { isElementOfType, isNodeOfType } from 'roosterjs-content-model-dom';
+import type {
+    ContentModelEntity,
+    EntityInfoFormat,
+    FormatParser,
+    OnNodeCreated,
+} from 'roosterjs-content-model-types';
+
+const BLOCK_ENTITY_CLASS = '_EBlock';
+const ONE_HUNDRED_PERCENT = '100%';
+
+/**
+ * @internal
+ */
+export const onCreateCopyEntityNode: OnNodeCreated = (model, node) => {
+    if (
+        (model as ContentModelEntity)?.wrapper &&
```

Comment:
better also check segmentType in case other object type can have wrapper in the future

Reply:
added the segment check here too. Thanks